src/main/java/com/hackathonorganizer/hackathonwriteservice/hackathon/model/Hackathon.java

```diff
@@ -1,36 +1,71 @@
 package com.hackathonorganizer.hackathonwriteservice.hackathon.model;
 
+import com.fasterxml.jackson.annotation.JsonFormat;
 import com.hackathonorganizer.hackathonwriteservice.team.model.Team;
-import lombok.Getter;
-import lombok.Setter;
+import lombok.*;
+import lombok.extern.slf4j.Slf4j;
 
 import javax.persistence.*;
 import javax.validation.constraints.NotEmpty;
 import javax.validation.constraints.NotNull;
 import java.time.LocalDateTime;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 
 @Entity
 @Getter
 @Setter
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@Slf4j
 public class Hackathon {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @NotEmpty
+    private String name;
+
     @NotEmpty
     private String description;
 
     @NotEmpty
     private String organizerInfo;
 
+    @Builder.Default
+    private boolean isActive = true;
+
     @NotNull
+    @JsonFormat(pattern = "HH:mm:ss dd-MM-yyyy")
     private LocalDateTime eventStartDate;
 
     @NotNull
+    @JsonFormat(pattern = "HH:mm:ss dd-MM-yyyy")
     private LocalDateTime eventEndDate;
 
     @OneToMany(mappedBy = "hackathon")
     private List<Team> teams;
+
+    @ElementCollection
+    @CollectionTable(name = "hackathon_participants", joinColumns =
```

Comment:
Formatowanie, troche inne ustalilsmy, zobacz na user-service

Reply:
@misterrevo To wyeksportuj i wrzuc gdzies te ustawienia z intelij